packages/data-point/lib/entity-types/entity-request/resolve.js

```diff
@@ -41,10 +41,10 @@ module.exports.getRequestOptions = getRequestOptions
  * @return {Promise<Accumulator>}
  */
 function resolveOptions (accumulator, resolveReducer) {
-  accumulator = resolveUrl(accumulator)
+  const url = resolveUrl(accumulator)
   const specOptions = accumulator.reducer.spec.options
   return resolveReducer(accumulator, specOptions).then(acc => {
-    const options = getRequestOptions(acc.url, acc.value)
```

Comment:
adding url to the accumulator was wrong from the beginning, this change also improves speed (very little but it does) since we no longer use `utils.assign`